src/app/services/standup.ts

```diff
@@ -0,0 +1,15 @@
+import { standupUpdateType } from '@/interfaces/standup.type';
+import { api } from './api';
+
+export const standupApi = api.injectEndpoints({
+    endpoints: (build) => ({
+        addStandUp: build.mutation<void, standupUpdateType>({
```

Comment:
addStandUp->addStandup

as I am seesing you write everywhere like this

Reply:
fixed